CHANGELOG.md

```diff
@@ -1,13 +1,15 @@
 ## Unreleased
 
 ## 0.13.2 - 2018-05-08
+- Adds Tracing.getExportComponent().shutdown() for use within application shutdown hooks.
```

Comment:
0.13.2 was already released, so this should go under unreleased.

Reply:
ugh, sorry, wasn’t paying attention. fixed.